aws/cft.yml

```diff
@@ -0,0 +1,141 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Conjur CE Stack, backed by a PostgreSQL database on RDS
+
+Parameters:
+  ConjurAccount:
+    Type: String
+    Description: Name of Conjur account
+
+  ConjurAdminPassword:
+    Type: String
+    NoEcho: true
+    Description: Password for Conjur admin account
+
+  DBAdminUser:
+    Type: String
+    Description: RDS instance admin user
+    Default: conjuradmin
```

Comment:
This is often a sensitive parameter as well, can we mark it `NoEcho`?

Reply:
The problem I had with marking this `NoEcho` is that I couldn't find a way to ask the user to enter it a second time, and then verify that the two matched.

Only being able to enter it once, and not being able to see it, seems like a recipe for rendering it unusable (when you fat-finger the password, like I always do).